resources/js/pages/achievement/[achievement]/comments.tsx

```diff
@@ -10,17 +10,13 @@ const AchievementComments: AppPage<App.Community.Data.AchievementCommentsPagePro
 }) => {
   const { t } = useTranslation();
 
-  const metaDescription = `General discussion about the achievement ${achievement.title}`;
-
   return (
     <>
-      <Head title={t('Comments - {{achievementTitle}}', { achievementTitle: achievement.title })}>
-        <meta name="description" content={metaDescription} />
-        <meta name="og:description" content={metaDescription} />
-
-        <meta property="og:image" content={achievement.badgeUnlockedUrl} />
-        <meta property="og:type" content="retroachievements:comment-list" />
```

Comment:
All of these og:type values have been replaced with "website".

User-defined types like "retroachievements:comment-list" are valid, but not officially recognized. The valid types can be found here: https://ogp.me/#no_vertical